src/components/TaxWidget.jsx

```diff
@@ -355,7 +355,7 @@ class TaxWidget extends Component {
           </div>
           <div className="exchanges">
             <div className="heading">
-              <h2>Choose Exchanges</h2>
+              <h2>Choose Marketplace</h2>
```

Comment:
Left singular on purpose. Right now we support only one exchange. Also you choose one market at a time thought at the end you might have couple selected.